### PR TITLE
missing space & wrong tree

### PR DESCRIPTION
**Pull request by bitjimy**
missing space in list, and tree for ((B C) C) and not ((A B) C)

### CODE REVIEW

**Comment by gur111** [comment]
Please merge with master to solve conflicts

**Review by gur111 — COMMENTED** [review]
Conflicts. Yes, I know, it's not your fault. But please fix conflicts and I'll CR and merge

**Comment by bitjimy** [comment]
> Conflicts. Yes, I know, it's not your fault. But please fix conflicts and I'll CR and merge

done.

File: lisp.tex [hunk 1]
```diff
@@ -740,8 +740,8 @@
   ✦ האטום \T|atom| מציין פונקציה המקבלת ביטוי~S אחד ומחזירה את האטום \T|t| אם
   ארגומנט זה הוא אטום, ואחרת את האטום \T|nil|.
 
-  ✦ האטום \T|null| מציין את הפונקציה המקבלת ביטויי~S אחד ומחזירה את האטום
-  \T|t|. אם ארגומנט זה הוא האטום \T|nil| ואחרת את האטום \T|t| אם הארגומנט
+  ✦ האטום \T|null| מציין את הפונקציה המקבלת ביטוי~S אחד ומחזירה את האטום
+  \T|t| אם ארגומנט זה הוא האטום \T|nil| ואחרת את האטום \T|nil| אם הארגומנט
```
Comment (gur111) [inline lisp.tex:744]: Add missing dot

File: lisp.tex [hunk 2]
```diff
@@ -763,11 +763,11 @@
 בהנתן אלפבית~$Σ$, נגדיר את~$T(Σ)$, קבוצת העצים שבכל צומת שלהם ישנו איבר של~$Σ$.
 עץ כזה יכול להיות עץ אטומי, ואז העץ מצטמצם לכדי עלה בודד, שחייב להכיל איבר
 של~$Σ^*$. עץ ב-$T(Σ)$ יכול להיות גם עץ מורכב, ובמקרה זה הוא מורכב מצומת פנימי
-שבו יש אות של~$Σ$ ומספר כלשהו של בנים שכולם עצים ב-\E|$T(S)$|. בניסוח זה, אנו
+שבו יש אות של~$Σ$ ומספר כלשהו של בנים שכולם עצים ב-\E|$T(Σ)$|. בניסוח זה, אנו
 רואים שעץ אטומי הוא עץ מורכב שיש לו אפס בנים.
 
 נוכל להגדיר את הקבוצה~$T(Σ)$ כשפה פורמלית מעל אלפבית מורחב, הכולל גם את זוג
-סימני הסוגריים ואת סימן הפסיק בא בהנתן אלפבית~$Σ$ אזי,~$T(Σ)$,
+סימני הסוגריים ואת סימן הפסיק בא בהנתן אלפבית~$Σ$ אזי,~$T(Σ)$.
```
Comment (gur111) [inline lisp.tex:770]: Not sure I agree with that fix, @yossigil ?

File: lisp.tex [hunk 3]
```diff
@@ -1077,7 +1077,7 @@
 אם נכתוב כעת
 \begin{LISP}
 > (set (quote foo) (quote ((baz))))
-(BAR BAZ)
+((BAZ))
```
Comment (gur111) [inline lisp.tex:1080]: @yossigil ?

File: lisp.tex [hunk 4]
```diff
@@ -1119,7 +1119,7 @@
 ואם ננסה לשערך כעת את \T|foo|, נקבל את הערך הנוצר מהקישור החדש
 \begin{LISP}
 > foo
-((BAZ) BAR)
+((BAZ))
```
Comment (gur111) [inline lisp.tex:1122]: @yossigil ?

**Review by gur111 — CHANGES_REQUESTED** [review]
Let's just wait for Yossi's approval on the ones I'm not sure.
Meanwhile fix the dot thing.

**Comment by yossigil** [comment]
I am sorry, but the dev version has converged, and there are still conflicts. Can you please create a new PR with respect to the current dev version.

**Pull request closed without merging**